models/metrics/actions/actions_proc.sql

```diff
@@ -65,9 +255,8 @@ CASE WHEN http_status_code != 200 OR flag_paginated = 1 OR canonical_status = 'c
 	ELSE '' END as cannibalization_action,
 
 CASE WHEN http_status_code != 200 OR flag_paginated = 1 OR canonical_status = 'canonicalized' THEN '' 
-	WHEN http_status_code = 200 AND sessions_mom_pct > total_organic_sessions_mom_pct and ctr_mom_pct > total_ctr_mom_pct THEN 'Rising content'
-	WHEN http_status_code = 200 AND transactions_30d > 0 and sessions_30d > med_sessions_30d and ecommerce_conversion_rate_mom_pct < total_organic_ecommerce_conversion_rate_mom_pct THEN 'Below-trend conversion rate'
-	WHEN http_status_code = 200 AND sessions_30d > med_sessions_30d and goal_conversion_rate_all_goals_mom_pct < total_organic_goal_conversion_rate_mom_pct THEN 'Below-trend conversion rate'
+	WHEN http_status_code = 200 AND sessions_30d != 0 AND sessions_30d_mom != 0 AND (sessions_30d - sessions_30d_mom)/sessions_30d > 0.1 THEN 'Rising content'
+	WHEN http_status_code = 200 AND sessions_30d != 0 AND sessions_30d_mom != 0 AND (sessions_30d - sessions_30d_mom)/sessions_30d < -0.1 THEN 'failing_content'
```

Comment:
Let's call this status 'Falling content' rather than 'failing_content' (ie rising as opposite of falling).